stylesheets/styles.css

```
@@ -333,11 +333,17 @@ section {
 /* Footer */
 
 footer {
-  width:170px;
+  margin: 0 auto;
+  width: 960px;
   float:left;
   position:fixed;
-  bottom:10px;
-	padding-left: 50px;
+  bottom:0px;
+  padding-left: 50px;
+  background-color: #FBFAF7;
+}
+
+footer p {
+    margin: 0 0 0px;
```

margin のインデントが深くなっているのと、0 の指定は ↑→↓←があるので 3 つよりは 4 つのほうがわかりやすいんじゃないかな、と思いました。

```suggestion
  margin: 0 0 0px;
```

---

> margin のインデントが深くなっている
[cc5763f] こちらはこのコミットで

> 0 の指定は ↑→↓←があるので 3 つよりは 4 つのほうがわかりやすい
[2631c15] こちらはこのコミットで

修正をしました🙇‍♂️！